minikube-gradle-plugin/README.md

```diff
@@ -30,7 +34,20 @@ minikubeStart {
 }
 ```
 
-This plugin also allows users to add in any custom minikube task.
+`minikubeDockerBuild` task is of the type `DockerBuildTask` and can be configured with:
+- `context` (`String`) : PATH | URL | - (See 'Extended description' under the [`docker build` Reference](https://docs.docker.com/engine/reference/commandline/build/))
+- `flags` (`String[]`) : any flags to pass to `docker build` (See 'Options' under the [`docker build` Reference](https://docs.docker.com/engine/reference/commandline/build/))
+- `minikube` (`String`) : path to minikube executable which should be set by using the `minikube` extension **users should not edit this for this provided task**
+- `docker` (`String`) : path to Docker executable which should be set by using the `docker` extension **users should not edit this for this provided task**
+
+```groovy
+minikubeDockerBuild {
+  context = "."
```

Comment:
I wonder if we can be more helpful here for build, providing reasonable defaults.
1. `dockerfile` param in minikubeDockerBuild which defaults to `src/main/docker/Dockerfile` and configures the task with `-f src/main/docker/Dockerfile`. Since we don't really expect to find Dockerfile in the root of a gradle project.
2. context will probably be in `build/libs/` but there is a way to programmatically extract the final artifact with gradle (not sure if this is what all users want), but we can start with setting the context default to `build/libs`.

This can be in a follow-up PR - and just move what you want into a new issue and start from there.

Reply:
Also more context for using src/main/docker <--- seems to be the default for a lot of other gradle plugins (including out gradle plugin).

Reply:
Okay I have set the default context to be `build/libs`. I'll file a PR about providing more help for the build.